src/components/test/test.css

```diff
@@ -264,3 +264,9 @@
   max-width: 100%;
   height: auto;
 }
+
+.video {
+  display: block;
```

Comment:
Same as img CSS. Makes the video scale responsively like the image tag does.

Reply:
This should be combined with the `.image` class definition since they are the same.